exercises/house/HINTS.md

```diff
@@ -0,0 +1,18 @@
+## Hints
+
+This exercise is about [code refactoring](https://en.wikipedia.org/wiki/Refactoring),
+so we are providing you with a solution that already passes the tests.
+
+The challenge is to rewrite it until you are proud of it, and learn
+something in the process.
+
+If you don't know where to start, here are some ideas:
+
+- Try to reduce repetition to a minimum.
+- Try to make the code readable.
+
+Take your time.
```

Comment:
https://www.youtube.com/watch?v=RuqaVryDRd0&t=10

Ignore this comment, it is not serious.